integration/sandbox_run_rollback_linux_test.go

```diff
@@ -84,19 +92,293 @@ func TestRunPodSandboxWithShimStartFailure(t *testing.T) {
 	require.Equal(t, true, strings.Contains(err.Error(), "no hard feelings"))
 }
 
+// TestRunPodSandboxWithShimDeleteFailure should keep the sandbox record if
+// failed to rollback shim by shim.Delete API.
+func TestRunPodSandboxWithShimDeleteFailure(t *testing.T) {
+	if runtime.GOOS != "linux" {
```

Comment:
since I use _linux_test.go to name this file, we don't need this.

Reply:
nit: these removals (and the `ENABLE_CRI_SANDBOXES` removals) were done in the "Update container with sandbox metadata after NetNS is created" commit.  Can you move them into this ("Add integration tests with failpoint") commit instead so that they don't appear in the branch history?